operatorcourier/validate.py

```diff
@@ -237,3 +245,135 @@ def _type_validation(self, bundleData, typeName, validator, required):
             valid = False
 
         return valid
+
+    def _ui_validation(self, bundleData):
+        valid = True
+        logger.info("Validating cluster service versions for operatorhub.io UI.")
+
+        csvs = bundleData[self.csvKey]
+
+        for csv in csvs:
+            if self._ui_csv_fields_exist_validation(csv) is False:
+                logger.error("UI validation failed to verify required fields exist.")
+                valid = False
+            
+            if valid:
+                if self._ui_csv_fields_format_validation(csv) is False:
+                    logger.error("UI validation failed to verify required fields exist.")
```

Comment:
Can this error be more descriptive? Getting here requires that fields exist, but isn't formatted properly. Is that correct?

Reply:
This is the general error that will be displayed after the validation function returns. The exact errors that caused validation to fail will be listed before the above error.